src/ui/public/filter_bar/lib/disable_filter.js

```diff
@@ -0,0 +1,25 @@
+export function disableFilter(filter) {
+  return setFilterDisabled(filter, true);
+}
+
+export function enableFilter(filter) {
+  return setFilterDisabled(filter, false);
+}
+
+export function setFilterDisabled(filter, disabled) {
+  const { meta = {} } = filter;
+
+  return {
```

Comment:
I'm not sure I buy it, but it sounds like a debate not worth getting into here :)   let's leave it as is.